core/core/sustainability_labels.py

```diff
@@ -0,0 +1,117 @@
+    certificate_information[certificate].update(evaluation)
+
+certificate_information.update(special_labels)
+
+# Create a mapping from the full certificate name to its id without the certificate_tag
+_certificate_2_id = {
+    certificate.replace(certificate_prefix, ""): certificate
+    for certificate in certificate_information.keys()
+}
+
+
+def get_CertificateType_enum() -> EnumMeta:
+    """
+    Factory function to construct the `Enum` that contains the sustainable certificates
+
+    Returns:
+        EnumMeta: `CertificateType` `Enum` use for the domain.
+    """
+
+    class CertificateHelper(Enum):
+        """
+        We use the `Enum` Functional API: https://docs.python.org/3/library/enum.html#functional-api
+        to construct an Enum based on the sustainability certificates. For this, we need to
+        implement `_generate_next_value_`, however, this class is not used any further.
+        """
+
+        def _generate_next_value_(name: str, start, count, last_values):  # type: ignore
+            return _certificate_2_id[name]
+
+    return CertificateHelper(  # type: ignore
+        value="CertificateType",
+        names=list(_certificate_2_id.keys()),
+        module="core.domain",
+        type=str,
+    )
+
+
+def _get_certificate_attribute(
+    certificate_information_dict: Dict[str, Dict[str, Any]],
+    attribute: str,
+    language_preference: List[str] = ["de", "en", "fr"],
+) -> str:
+    """
+    Helper function to retrieve an attribute of a certificate_information_dict in one language.
+
+    Args:
+        certificate_information_dict (Dict[str, Dict[str, Any]]): Dictionary that holds the
+            sustainability label information.
+        attribute (str): `attribute` to get.
+        language_preference (List[str], optional): Order of language priorities.
+            Defaults to ["de", "en", "fr"].
+
+    Returns:
+       str: of the corresponding attribute in one language.
+    """
+    for language in language_preference:
+        if language in certificate_information_dict["languages"].keys():
+            return certificate_information_dict["languages"][language][attribute]
+    return ""
+
+
+# Create a list of SustainabilityLabel with its information in one language
+certificate_information_dense = [
+    SustainabilityLabel(
+        id=certificate_id,
+        timestamp=datetime(2022, 3, 3, 19),  # NOTE: Change me after updating sustainability labels
+        name=_get_certificate_attribute(certificate_information_dict, "name"),
+        description=_get_certificate_attribute(certificate_information_dict, "description"),
+        ecological_evaluation=certificate_information_dict.get("ecological_evaluation", None),
+        social_evaluation=certificate_information_dict.get("social_evaluation", None),
+        credibility_evaluation=certificate_information_dict.get("social_evaluation", None),
+    )
+    for certificate_id, certificate_information_dict in certificate_information.items()
+]
```

Comment:
I recognized this one does not work. Since it is important to keep `SustainabilityLabel` in `core.domain`!
Maybe we need another file? WDYT?

Reply:
Yes, I also had the circular import problem, thats why I moved the class out of `core.domain`. And I did not expect it to be a problem, because now all the sustainability related information is in one place in `core.sustainability_labels` . But if you want to keep it in the `core.domain` we could create another file like you proposed. Do you have a sensible filename in mind?